joomla/joomla-cms
# Move resources.joomla.org to HTTPS

### Summary of Changes

Move resources.joomla.org to HTTPS

### Testing Instructions

- Install: https://github.com/zero-24/joomla-cms/archive/jrd_https.zip
- confirm that the JRD links are HTTPS and not longer HTTP

### Expected result

resources.joomla.org on HTTPS

### Actual result

resources.joomla.org on HTTP

### Documentation Changes Required

None
